dynamic.js

```diff
@@ -209,6 +209,12 @@ module.exports = function (fastify, opts, next) {
         if (schema.security) {
           swaggerMethod.security = schema.security
         }
+
+        Object.keys(schema).forEach(key => {
```

Comment:
would you mind to use a for...if loop instead?

Reply:
It should be fixed now